### PR TITLE
(DOCSP-38554): Add a Monitor Edge Server page

### PR DESCRIPTION
**Pull request by dacharyc**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-38554

- [Monitor Edge Server](https://preview-mongodbdacharyc.gatsbyjs.io/atlas-app-services/DOCSP-38554/edge-server/monitor/): New page with information about Edge Server logs, metrics, and Edge Server status.

Todos to carry over to feature branch:

- [ ] Populate Admin API details

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [x] Did you tag pages appropriately?
  - genre
  - programming_language
  - meta.keywords
  - meta.description
- [x] Describe your PR's changes in the Release Notes section
- [x] Create a Jira ticket for related docs-realm work, if any

### Release Notes

- **Edge Server**
  - Monitor Edge Server: New page with information about Edge Server logs, metrics, and status.

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/edge-server/monitor.txt [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+You can access your metrics data with the Admin API's
+[PLACEHOLDER - insert Admin API details here] endpoint. Or you can view a
+subset of these metrics when viewing :ref:`edge-server-details` in the 
+Atlas UI.
+
+The exact metrics returned may be subject to change.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 15 15 25 15 25
```
Comment (dacharyc) [inline source/edge-server/monitor.txt:170]: Note for reviewer: this table is annoyingly truncating the columns and does not seem to be respecting the `widths` I am setting here. I'll look into this next week.

File: source/edge-server/monitor.txt [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+
+Edge Server Logs
+----------------
+
+Edge Server uses the following log types:
+
+- :ref:`authentication-logs`: When an Edge Server instance attempts to 
+  authenticate with Atlas, or clients attempt to authenticate with an Edge
+  Server instance, this generates an authentication log.
+- :ref:`logs-sync`: When an Edge Server instance syncs with Atlas, or an
```
Comment (sudssm) [inline source/edge-server/monitor.txt:131]: starting monday, you'll also see logs when device connect to and auth with the edge server, provided that the edge server is able to send those logs up

File: source/includes/edge/edgectl-status.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+     - The current version of the Edge Server code that the instance is 
+       running. If the Edge Server instance falls more than one major version
+       behind the latest available version, it is unable to connect to Atlas
+       and must be wiped and updated. For more information, refer to 
+       :ref:`upgrade-edge-server-version`.
+
+   * - ``status``
+     - String
+     - The current status of the Edge Server instance. Possible values include:
+       ``"ACTIVE"``, ``"FAILED"``.
```
Comment (sudssm) [inline source/includes/edge/edgectl-status.rst:62]: there's also BOOTSTRAPPING, which happens when the edge server is making its initial download and not yet ready to use

**Review by cbullinger — APPROVED** [review]

File: source/edge-server/monitor.txt [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+      :tabid: atlas-ui
+
+      You need the following to manage Edge Servers in the Atlas UI:
+
+      - .. include:: /includes/prereqs/atlas-account.rst
+   
+   .. tab:: Local UI
+      :tabid: local-ui
+
+      The local UI is enabled by default, and only requires a running
```
[the remark this answers is not in the document]
Comment (dacharyc) [reply source/edge-server/monitor.txt:49]: We do not, aside from listing the flag to disable it in the configuration options. As the local UI is really about monitoring the Edge Server instance, I didn't see any reason to bring it up on other pages.

[the remark this answers is not in the document]
Comment (cbullinger) [reply source/edge-server/monitor.txt:49]: not really important; it was just occurred to me that I didn't know there was a local ui until now (despite having seen the config option 😅 )

**Pull request merged**